build.sbt

```diff
@@ -88,6 +89,38 @@ lazy val warnUnusedImport = Seq(
   scalacOptions in (Test, console) <<= (scalacOptions in (Compile, console))
 )
 
+lazy val framelessTypedDatasetREPL = Seq(
+  initialize ~= { _ => // Color REPL
+    val ansi = System.getProperty("sbt.log.noformat", "false") != "true"
+    if (ansi) System.setProperty("scala.color", "true")
+  },
+  initialCommands in console :=
+    """
+      |import org.apache.spark.{SparkConf, SparkContext}
+      |import org.apache.spark.sql.SparkSession
+      |import frameless.functions.aggregate._
+      |
+      |val conf = new SparkConf().setMaster("local[*]").setAppName("frameless repl").set("spark.ui.enabled", "false")
+      |val spark = SparkSession.builder().config(conf).appName("REPL").getOrCreate()
+      |
+      |import spark.implicits._
+      |
+      |spark.sparkContext.setLogLevel("WARN")
```

Comment:
I cleaned it up a bit so that it resembles more to spark. I didn't include all those imports. I think we can do this as we see fit (e.g., most of the time I don't use standard spark ops, I use the ones we provide from frameless.